### ISSUE TITLE
bug: blank character's `rank` in `guild/{name}` request

### ISSUE THREAD

**Issue by rmobis** [TibiaData/tibiadata-api-go#424]
For some reason when picking up the rank for the leader of the guild _True Players_, which should be _We_, it returns a blank string.

TibiaData URL:
https://api.tibiadata.com/v4/guild/True%20Players

Tibia URL:
https://www.tibia.com/community/?subtopic=guilds&page=view&GuildName=True+Players

### Expected
```json
{
  "guild": {
    "name": "True Players",
    "world": "Karmeya",
    // ...
    "members": [
      {
        "name": "Loo Mind Picture",
        "title": "",
        "rank": "We",
        "vocation": "Master Sorcerer",
        "level": 605,
        "joined": "2024-11-24",
        "status": "online"
      }
      // ...
    ]
  }
}
```

### Got
```json
{
  "guild": {
    "name": "True Players",
    "world": "Karmeya",
    // ...
    "members": [
      {
        "name": "Loo Mind Picture",
        "title": "",
        "rank": "",
        "vocation": "Master Sorcerer",
        "level": 605,
        "joined": "2024-11-24",
        "status": "online"
      }
      // ...
    ]
  }
}
```

### Tibia Website
![image](https://github.com/user-attachments/assets/0fcad4f9-04cd-4baa-b6a8-bb7429a62278)

**Comment by tobiasehlert** [comment]
Thanks for reporting this @rmobis :)

**Comment by tobiasehlert** [comment]
@rmobis, this is fixed with release [v4.3.2](https://github.com/TibiaData/tibiadata-api-go/releases/tag/v4.3.2).
